pkg/openapi/swagger.json

```diff
@@ -1169,7 +1169,7 @@
       "type": "object",
       "properties": {
         "annotations": {
-          "description": "Annotations that will be add to the component pod. More info: http://kubernetes.io/docs/user-guide/annotations",
+          "description": "Annotations that will be added to the component pod. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/",
```

Comment:
Wondering how was the CI didn't complain about this ? Maybe not rebased ? Should we have a check for rebase ?

Reply:
Oh Okay. Also I am thinking we can add a auto retry mechanism for Infra setup actions.https://github.com/marketplace/actions/retry-action. WDYT ?

Reply:
Sounds good